slack-glickman/commands/game_logs.rb

```diff
@@ -76,7 +76,7 @@ class GameLogs < SlackGlickman::Commands::BaseCommand
           game_logs = const_get("Stattleship::#{sport.capitalize}GameLogs").fetch(params: query_params)
 
           if game_logs.size > 0
-            msg = game_logs.first.to_sentence
+            msg = game_logs.max_by { |log| log.game.timestamp }.to_sentence
```

Comment:
I've never used `max_by` before; I like it.

Way better than sort and first or limit 1.